prisma/schema.prisma

```diff
@@ -24,24 +24,25 @@ enum CuratedItemSource {
 // and (optionally) making it to a Scheduled Surface.
 model ApprovedItem {
   // fields
-  id              Int           @id @default(autoincrement())
-  externalId      String        @default(uuid()) @db.VarChar(255)
-  prospectId      String?       @db.VarChar(255)
-  url             String        @db.VarChar(500)
-  title           String        @db.VarChar(255)
-  excerpt         String        @db.Text
-  status          CuratedStatus @default(CORPUS)
-  language        String        @db.VarChar(2)
-  publisher       String        @db.VarChar(255)
-  imageUrl        String        @db.VarChar(500)
-  topic           String        @db.VarChar(255)
-  isCollection    Boolean       @default(false)
-  isTimeSensitive Boolean       @default(false)
-  isSyndicated    Boolean       @default(false)
-  createdAt       DateTime      @default(now())
-  createdBy       String        @db.VarChar(255)
-  updatedAt       DateTime      @updatedAt
-  updatedBy       String?       @db.VarChar(255)
+  id              Int                @id @default(autoincrement())
+  externalId      String             @default(uuid()) @db.VarChar(255)
+  prospectId      String?            @db.VarChar(255)
+  url             String             @db.VarChar(500)
+  title           String             @db.VarChar(255)
+  excerpt         String             @db.Text
+  status          CuratedStatus      @default(CORPUS)
+  language        String             @db.VarChar(2)
+  publisher       String             @db.VarChar(255)
+  imageUrl        String             @db.VarChar(500)
+  topic           String             @db.VarChar(255)
+  source          CuratedItemSource  @default(PROSPECT)
```

Comment:
Is this the correct default?

Reply:
`prospect` as default value for `source` would work in automatedmutation case as we are explicitly setting this value

but if a curator adds an item manually, how will the `createApprovedItem` mutation will set this to `manual`?

Reply:
this new field will require changes in the curation tool. i'll make a ticket for that now.

Reply:
- we can make the `source` as nullable field without default to unblock the PR
- we will have follow up ticket (Created by jonathan) to add `source` to other mutations
- then we set a default to source (Depending on product team needs) - but this can happen outside this work

Reply:
implemented per @sri-kirsh's comment above